app/views/appointments/new.html.erb

```diff
@@ -55,7 +55,34 @@
         <div class="appointment">
           <input type="submit" value="Submit">
         </div>
-      </form>
+      </form> -->
+      <%= form_for @appointment do |f| %>
```

Comment:
PS: This is also true for messages, as only registered users can send messages, I'll add that to your issues